hoot-core/src/main/cpp/hoot/core/cmd/ConvertCmd.cpp

```diff
@@ -78,10 +82,25 @@ class ConvertCmd : public BaseCommand
     OsmMapReaderFactory readerFactory = OsmMapReaderFactory::getInstance();
     OsmMapWriterFactory writerFactory = OsmMapWriterFactory::getInstance();
 
+    if (args[0].endsWith(".osc"))
+    {
+      throw HootException("XML changeset file writing is not supported by the convert command.");
+    }
+    else if (args[0].endsWith(".osc.sql"))
```

Comment:
What is this doing? Did you just change the convert command to support a "apply changeset" operation too?